pkg/authorization/authorizer/scope/converter_test.go

```diff
@@ -122,7 +122,7 @@ func TestClusterRoleEvaluator(t *testing.T) {
 				},
 			},
 			scopes:   []string{ClusterRoleIndicator + "missing:*"},
-			err:      `clusterrole "missing" not found`,
+			err:      `clusterrole.authorization.openshift.io "missing" not found`,
```

Comment:
Because of: https://github.com/mfojtik/origin/blob/2259dd5022d7ff7af22d53d4d3214e935c19bdf5/pkg/authorization/authorizer/scope/converter.go#L330

I can change that to LegacyResource() to preserve legacy behavior. Not sure I will capture all errors like this everywhere as some of them are returned from kube helpers that calls `.Resource()`. Guess something we will have to live with till we figure out how to set "preferred resource name" properly in upstream.